WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1245,6 +1246,11 @@ public void onClick(View v) {
                     public void onClick(DialogInterface dialog, int which) {
                         String entry = input.getText().toString();
                         if (!TextUtils.isEmpty(entry) && !mEditingList.contains(entry)) {
+                            if (mEditingList == mSiteSettings.getJetpackWhitelistKeys() && !isValidIpOrRange(entry)) {
```

Comment:
`mEditingList` can actually be a reference to several lists (blacklist, whitelist, or moderation keys). It gets set in `onPreferenceClick` for those preferences that use the list editor dialog and is used to determine diff the changes made. Hopefully the comment I added plus the [comment at mEditingList's declaration](https://github.com/wordpress-mobile/WordPress-Android/blob/8fc1d8a433d4e7beeda4a7cb7ab8676bbe4be7a3/WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java#L146) provide enough information.